### PR TITLE
Upgrade declarative-option to support ruby >= 2.7

### PR DESCRIPTION
**Pull request by yogeshjain999**

### CODE REVIEW

File: test/prepare_test.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def ==(b)
   describe "#to_hash" do # TODO: introduce :representable option?
     representer! do
       property :song,
-        :prepare       => lambda { |options| options[:binding][:arbitrary].new(options[:input]) },
+        :prepare       => lambda { |_, binding:, input:, **| binding[:arbitrary].new(input) },
```
Comment (seuros) [inline test/prepare_test.rb:21]: don't use `binding` it a ruby keyword.

Comment (yogeshjain999) [reply test/prepare_test.rb:21]: Isn't that a method ?

File: test/wrap_test.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ class SoftcoreBand < HardcoreBand
       let(:format) { format }
 
       representer!(:module => mod) do
-        self.representation_wrap = lambda { |args| "#{name}#{args[:number]}" }
+        self.representation_wrap = lambda { |number:, **| "#{name}#{number}" }
```
Comment (yogeshjain999) [inline test/wrap_test.rb:39]: @apotonick I changed only few test expectations like this and one more to use kwargs, as there were none. 

https://github.com/trailblazer/representable/pull/240/files#diff-367119c875f06023f45c03f9dfefa7cf6d6dc761c50746e60e464f6548904cf1R6

**Comment by yogeshjain999** [comment]
Closing this according to https://github.com/trailblazer/representable/pull/246

**Pull request closed without merging**